arch/esp32/esp32.ini

```diff
@@ -37,6 +37,7 @@ build_flags =
   -DLIBPAX_ARDUINO
   -DLIBPAX_WIFI
   -DLIBPAX_BLE
+  -DHAS_UDP_MULTICAST=1
```

Comment:
absolutely ok for testing, later i would always enable that if we have an IP interface (wifi, eth)